max-model-status.md

```diff
@@ -49,6 +49,7 @@ Current status for the https://developer.ibm.com/exchanges/models/ ecosystem.
 
 | Name | Status | Latest Release |
 | ---- | ---- | ---- | 
+| [Question Answering Web App](https://github.com/IBM/MAX-Question-Answering-Web-App) | [![Website](https://img.shields.io/website/http/max-question-answering-web-app.max.us-south.containers.appdomain.cloud/app.svg)](http://max-question-answering-web-app.max.us-south.containers.appdomain.cloud)| [![GitHub release](https://img.shields.io/github/v/release/IBM/MAX-Question-Answering-Web-App.svg)](https://github.com/IBM/MAX-Question-Answering-Web-App/releases/latest) | 
```

Comment:
@ptitzler How do we get the website `svg` link?

Reply:
No build status?

Reply:
The badge link should be `https://img.shields.io/website/http/max-question-answering-web-app.max.us-south.containers.appdomain.cloud.svg`. However the q&a app itself has an issue that needs to be fixed to make this work. It's my understanding that shields.io runs the equivalent of `curl -I http://max-question-answering-web-app.max.us-south.containers.appdomain.cloud` which currently raises HTTP error 500. If you try this with our other deployed apps you should see that they all return a positive status code.

Reply:
Just open two issues in https://github.com/IBM/MAX-Question-Answering-Web-App

Reply:
https://github.com/IBM/MAX-Question-Answering-Web-App/issues/14